### PR TITLE
[v6r14] Adding BoincUserID (if available) to the job parameters.

### PR DESCRIPTION
**Pull request by CinziaLu**

### CODE REVIEW

File: WorkloadManagementSystem/JobWrapper/JobWrapper.py [hunk 1]
```diff
@@ -101,6 +102,7 @@ def __init__( self, jobID = None, jobReport = None ):
     self.sandboxSizeLimit = gConfig.getValue( self.section + '/OutputSandboxLimit', 1024 * 1024 * 10 )
     self.cleanUpFlag = gConfig.getValue( self.section + '/CleanUpFlag', True )
     self.pilotRef = gConfig.getValue( '/LocalSite/PilotReference', 'Unknown' )
+    self.userID = gConfig.getValue( '/LocalSite/BoincUserID', '' )
```
Comment (atsareg) [inline WorkloadManagementSystem/JobWrapper/JobWrapper.py]: Would be better to call this variable self.boincUserID, nobody knows what other userIDs we can have in the future

Comment (CinziaLu) [inline WorkloadManagementSystem/JobWrapper/JobWrapper.py]: Yes, done.

File: WorkloadManagementSystem/JobWrapper/JobWrapper.py [hunk 2]
```diff
@@ -101,6 +102,7 @@ def __init__( self, jobID = None, jobReport = None ):
     self.sandboxSizeLimit = gConfig.getValue( self.section + '/OutputSandboxLimit', 1024 * 1024 * 10 )
     self.cleanUpFlag = gConfig.getValue( self.section + '/CleanUpFlag', True )
     self.pilotRef = gConfig.getValue( '/LocalSite/PilotReference', 'Unknown' )
+    self.boincUserID = gConfig.getValue( '/LocalSite/BoincUserID', '' )
```
Comment (chaen) [inline WorkloadManagementSystem/JobWrapper/JobWrapper.py]: Just a naive question: doesn't ```0``` make more sense as a default value for a ID ? It is really just a detail

Comment (atsareg) [inline WorkloadManagementSystem/JobWrapper/JobWrapper.py]: Is BOINC user ID a numerical one ?

Comment (CinziaLu) [inline WorkloadManagementSystem/JobWrapper/JobWrapper.py]: Yes, it is.

**Comment by atsareg** [comment]
Review OK

**Pull request merged**